odoo/tools/view_validation.py

```diff
@@ -159,13 +159,13 @@ def decorator(fn):
 def relaxng(view_type):
     """ Return a validator for the given view type, or None. """
     if view_type not in _relaxng_cache:
-        with tools.file_open(os.path.join('base', 'rng', '%s_view.rng' % view_type)) as frng:
-            try:
-                relaxng_doc = etree.parse(frng)
-                _relaxng_cache[view_type] = etree.RelaxNG(relaxng_doc)
-            except Exception:
-                _logger.exception('Failed to load RelaxNG XML schema for views validation')
-                _relaxng_cache[view_type] = None
+        fileRngPath = os.path.join(os.path.dirname(os.path.relpath(__file__)), '..', 'addons', 'base', 'rng', '%s_view.rng' % view_type)
```

Comment:
That seems like an issue with the local setup somehow: `file_open` specifically tries to resolve paths from a module. Hand-resolving paths is incorrect.

Reply:
I note this comment !